nc-news/src/components/Articles.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect, useState } from "react";
+import { getArticles } from "../api";
+
+export default function Articles() {
+	const [articles, setArticles] = useState([]);
+	const [loading, setLoading] = useState(true);
+	useEffect(() => {
+		setLoading(loading);
+		getArticles()
+			.then((res) => {
+				setArticles(res);
+			})
+			.then(() => {
+				setLoading(!loading);
+			});
+	}, []);
+
+	return loading ? (
+		<section id="loading">
+			<h1>Loading</h1>
```

Comment:
The h1 is a little large on screen! Personal preference really so I'll leave it up to you but in case you hadn't seen it! You can throttle your internet speed to slow 3g if you are wondering how to view it.